standards/cryptography.md

```diff
@@ -0,0 +1,57 @@
+---
+expires: 2019-03-15
+---
+# Cryptography
+
+## The base principle
+
+All data **must** employ adequate and proportionate cryptography to preserve confidentiality and integrity whether data is at-rest or in-transit.
+
+## In-transit
+
+In-transit encryption techniques can both protect data during transit through cryptography but also help facilitate the establishing of identity of devices on one or more sides of the connection.
+
+### Transport Layer Security (TLS)
+
+The [National Cyber Security Centre (NCSC)](https://www.ncsc.gov.uk/) have published information on good TLS configurations [https://www.ncsc.gov.uk/guidance/tls-external-facing-services](https://www.ncsc.gov.uk/guidance/tls-external-facing-services).
+
+In general, subject to document exceptions (such as end-user needs and required legacy backwards compatiability)
+
+#### Testing
+
+Tools such as [Qualys SSL Server Test](https://www.ssllabs.com/ssltest/) and Check TLS services from [checktls.com](http://www.checktls.com/index.html) **should** be used where applicable to help identify most common issues and configuration problems.
```

Comment:
I would suggest *must* be used to validate that the implementation has been done correctly, and then *should* be regularly revalidated.

Reply:
PR inbound